src/cloudai/schema/test_template/jax_toolbox/slurm_command_gen_strategy.py

```diff
@@ -256,18 +226,13 @@ def generate_full_srun_command(
         """
         self._create_run_script(slurm_args, env_vars, cmd_args, extra_cmd_args)
 
-        start_container_run = str(cmd_args.get("load_container", "False")).lower() in ("true", "1", "yes")
+        start_container_run = cmd_args.get(f"{self.test_name}.load_container", False)
```

Comment:
Why is the self.test_name getting appended here? This is causing a bug in the generated sbatch script.

```
[cmd_args]
docker_image_url = "xxxxxx"
fdl_config = "paxml.contrib.gpu.scripts_gpu.configs.Synthetic5B"
load_container = true
```

ideally we should be able to parse this with `cmd_args.get("load_container", "False"). During parsing the cmd_args actually appends the test_name before it.

Reply:
Legacy from the previous approach. I've updated logic so we can access this key directly.